tests/nodeos_multiple_version_protocol_feature_test.py

```diff
@@ -142,16 +169,25 @@ def areNodesInSync(nodes:[Node]):
     # Therefore, 1st node will be out of sync with 2nd, 3rd, and 4th node
     # After a round has passed though, 1st node will realize he's in minority fork and then join the other nodes
     # Hence, the PREACTIVATE_FEATURE that was previously activated will be dropped and all of the nodes should be in sync
+    Utils.Print("+++ 1st Node should contain PREACTIVATE FEATURE +++")
     setValidityOfActTimeSubjRestriction(newNodes[1], "PREACTIVATE_FEATURE", False)
     setValidityOfActTimeSubjRestriction(newNodes[2], "PREACTIVATE_FEATURE", False)
 
     waitUntilBeginningOfProdTurn(newNodes[0], "defproducera")
-    newNodes[0].activatePreactivateFeature()
+    # Retry activatePreactivateFeature for the 1st node after it enters production window
+    for i in range(3):
+        newNodes[0].activatePreactivateFeature()
```

Comment:
need to move line 176 here so that it waits for the start of the prod turn each time it tries, and put a log statement before it so that we can identify that we have gone through this loop multiple times

Reply:
Fixed.